docs/source/conf.py

```diff
@@ -18,18 +18,18 @@
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-#sys.path.insert(0, os.path.abspath('.'))
+# sys.path.insert(0, os.path.abspath('.'))
```

Comment:
No need to run black on this file, I think it's autogenerated and hasn't been changed since.

Reply:
We restrict black to run on the /src directory so far to avoid this. Therefore I agree with Tiberiu here.

Reply:
gotcha. reverted in 7b40aa94c01d58b4dc27dc7b2cec55c15a1c1b7d